pulsar-functions/proto/src/main/proto/Function.proto

```diff
@@ -165,6 +165,7 @@ message SourceSpec {
     bool cleanupSubscription = 11;
     SubscriptionPosition subscriptionPosition = 12;
     uint64 negativeAckRedeliveryDelayMs = 13;
+    bool skipToLatest = 14;
```

Comment:
@nlu90 - was this discussed on the mailing list? This is a protobuf change. We have a general rule that those go through the PIP process. While there could be exceptions, we should always post about these kinds of changes on the mailing list.

Reply:
@michaeljmarshall As we discussed via slack, I'll email the mailing list about the function proto change.
But it should be backward compatible and won't cause any issue for previous users.